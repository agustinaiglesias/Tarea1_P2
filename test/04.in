definirFecha 28/02/2022
siguienteDia 
fechaActual
fechaActualExtendida
Fin