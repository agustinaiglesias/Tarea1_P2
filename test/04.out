1>Fecha definida en forma exitosa.
2>Se aumentó un día.
3>La fecha actual es 1/3/2022.
4>La fecha actual es 1 de Marzo de 2022.
5>Fin.
